product/stream_processor/integrations/events/functions.py

```diff
@@ -0,0 +1,44 @@
+import re
+from typing import Any, Generator, Sequence, TypeVar
+from uuid import uuid4
+
+from product.stream_processor.integrations.events.constants import DEFAULT_EVENT_VERSION
+from product.stream_processor.integrations.events.models.input import AnyModel, Event, EventMetadata
+
+T = TypeVar('T')
+
+_exclude_underscores = r'(?!^)(?<!_)'
```

Comment:
I chose `functions` as a more narrow version of `utils`. Similar to `tests/.../data_builders.py` instead of `utils.py`. That way I know where utility functions are, instead of bin packing into `utils.py`.

Makes sense?